pkg/segment/writer/packer.go

```diff
@@ -672,6 +680,38 @@ func encJsonNumber(key string, numType SS_IntUintFloatTypes, intVal int64, uintV
 	return valSize
 }
 
+func (ss *SegStore) updateColValueSizeInAllSeenColumns(colName string, size uint32) {
+	currentSize, ok := ss.AllSeenColumns[colName]
+	if !ok {
+		if ss.RecordCount > 0 {
+			// Not the first record in the segment, so the col value size is not consistent.
```

Comment:
update the comment to something of ...
// column appearing first time in the middle of the wip, so past recNums will be filled with BackFill_CVAL_TYpe, so mark this as inconsistent